### ISSUE TITLE
No intellisense on 'Case-sensitive Journaled HFS+' volumes

### ISSUE THREAD

**Issue by keepitterron** [Microsoft/vscode#10916]
- VSCode Version: 1.4.0
- OS Version: Osx 10.11.6

Steps to Reproduce:

1. create a new image with `hdiutil create -type SPARSE -fs 'Case-sensitive Journaled HFS+' IMAGENAME` and mount it
2. create/copy a typescript project in it

Mac OS Extended (Journaled) images work just fine.
I use a case-sensitive image to deal with the issues with git and renaming files.

**Comment by sandy081** [comment]
@keepitterron Is it only intellisense not working? How about other language specific features like Syntax highlighting, Go to definitions etc?

**Comment by keepitterron** [comment]
Syntax highlighting seems to be working fine. Go to definitions or paths don't work at all.
Also, errors are not reported for `ts`. I opened the same file from outside and inside the image and the one outside reports `ts` and `tslint` errors, the one in the image disk only the `tslint` errors.

**Comment by keepitterron** [comment]
If I enable typescript tracing I get in the output console:
```
Sending request: open (0). Response expected: no. Current queue length: 0
Sending request: geterr (1). Response expected: no. Current queue length: 0
Sending request: quickinfo (2). Response expected: yes. Current queue length: 0
TypeScript Service: tried to cancel request with sequence number 2. But request got already delivered.
Response received: quickinfo (2). Request took 3547 ms. Success: false . Message: Error processing request. No Project.
```

**Comment by sandy081** [comment]
@keepitterron Thanks for providing the information.

**Comment by waderyan** [comment]
@dbaeumer do you think this is a tsserver issue?

**Comment by dbaeumer** [comment]
@waderyan yes it is and they have fixed it with 2.0.

**Comment by waderyan** [comment]
@keepitterron thank you for bringing this up. This is fixed for TypeScript 2.0. Closing for now unless it reappears after 2.0.
